corehq/apps/userreports/README.md

```diff
@@ -385,6 +385,76 @@ Variables can be any valid numbers (Python datatypes `int`, `float`, and `long`
 
 More examples can be found on practical [examples page](corehq/apps/userreports/examples/examples.md#evaluator-examples).
 
+#### Case --> Form Lookup
```

Comment:
Implemented all but custom 'latest' form property [here](https://github.com/dimagi/commcare-hq/pull/10989/files). Still work in progress, but this is good time to get feedback, both in terms of code and expression syntax. @czue